### DISCUSSION TITLE
Docker x64 image?

### DISCUSSION THREAD

**Discussion by Kofl** [TriliumNext/Notes#399] [Q&A / Support] [answered]
Hi,

maybe I a missing something, but it seems the docker images are only available for [arm64 and armv7](https://hub.docker.com/r/triliumnext/notes/tags?page_size=&ordering=&name=latest)?

Is that by intention?

Thanks

**Comment by mjh2901** [comment]
Same issue, There apears to be no AMD64 version
no matching manifest for linux/amd64 in the manifest list entries

**Comment by eliandoran** [comment]
Thanks for reporting, maybe something broke with the CI build. I will have a look this evening and get back to you.

**Comment by Kofl** [comment, marked as answer]
Thanks, v0.90.6-beta solved the issue.

**Reply by eliandoran** [reply]
@Kofl , glad to hear.

Thank you for the report!

**Comment by mjh2901** [comment]
Thanks, v0.90.6-beta solved the issue.
I literally took my docker-compose install and changed it from trilium to trilium next, picked up my database, and is up and running... almost seamless

services:
  trilium:
    image: triliumnext/notes:latest
    container_name: triliumnext_notes
    network_mode: bridge 
    ports:
      - 8081:8080
    volumes:
      - /docker/trilium_notes/trilium-data:/home/node/trilium-data
    restart: unless-stopped
